spec/service_consumer/pact_helper.rb

```diff
@@ -0,0 +1,63 @@
+require "pact/provider/rspec"
+require "webmock/rspec"
+require "factory_bot_rails"
+
+require ::File.expand_path("../../config/environment", __dir__)
+
+Pact.configure do |config|
+  config.reports_dir = "spec/reports/pacts"
+  config.include WebMock::API
+  config.include WebMock::Matchers
+  config.include FactoryBot::Syntax::Methods
+end
+
+WebMock.allow_net_connect!
+
+def url_encode(str)
+  ERB::Util.url_encode(str)
+end
+
+Pact.service_provider "Locations API" do
+  honours_pact_with "GDS API Adapters" do
+    if ENV["PACT_URI"]
+      pact_uri(ENV["PACT_URI"])
+    else
+      base_url = "https://pact-broker.cloudapps.digital"
+      path = "pacts/provider/#{url_encode(name)}/consumer/#{url_encode(consumer_name)}"
+      version_modifier = "versions/#{url_encode(ENV.fetch('PACT_CONSUMER_VERSION', 'branch-master'))}"
+
+      pact_uri("#{base_url}/#{path}/#{version_modifier}")
+    end
+  end
+end
+
+Pact.provider_states_for "GDS API Adapters" do
+  set_up do
+    ENV["OS_PLACES_API_KEY"] = "some_key"
+    ENV["OS_PLACES_API_SECRET"] = "some_secret"
+    Postcode.find_by(postcode: "SW1A1AA").destroy
```

Comment:
Does this always find a postcode? I imagine us getting some `Cannot call method :destroy on NilClass` type errors... might be safer to do:

```suggestion
    postcode = Postcode.find_by(postcode: "SW1A1AA")
    postcode.destroy unless postcode.nil?
```